src/controller.h

```diff
@@ -14,7 +14,7 @@
 #define US 1000000 // 1 million microseconds per second
 #define TICK_RATE 60 // Steps per second
 #define DELAY US/TICK_RATE // Delay in microseconds to achieve tick rate
-#define MOVE_RATE 12 // Steps per movement
+#define MOVE_RATE 6 // Steps per movement
```

Comment:
Can you add a todo here to make this non-constant? Eventually we'd want a thing that speeds up right?